packages/styles/scss/patterns/cards-without-images/_cards-without-images-group.scss

```diff
@@ -0,0 +1,120 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '../../components/link-with-icon/link-with-icon';
+
+@mixin themed-items {
+  .bx--card-link {
+    background-color: $ui-01;
+    outline: 1px solid $ui-background;
+    &__title {
+      color: $text-01;
+    }
+    &__content {
+      color: $text-02;
+    }
+    &__footer {
+      svg {
+        fill: $link-01;
+      }
+    }
+    &:last-child {
+      background-color: $ui-05;
+      .bx--card-link__title {
+        color: $ui-02;
+      }
+      .bx--card-link__footer {
+        svg {
+          fill: $inverse-support-04;
+        }
+      }
+    }
+  }
+}
+@mixin cards-without-images-group {
+  .#{$prefix}--cards-without-images-group {
+    padding-bottom: $carbon--layout-05;
+    &:last-child {
+      padding-bottom: 0;
+    }
+    &__title {
+      @include type-style('expressive-heading-02');
+
+      padding-bottom: $carbon--spacing-07;
+    }
+    &__cards {
+      display: flex;
+      flex-wrap: wrap;
+    }
+    .bx--card-link {
+      @include carbon--make-col(8, 8);
+
+      display: flex;
+      padding: $carbon--spacing-05;
+      text-decoration: none;
+      outline-offset: -1px;
+
+      > *:not(:last-child) {
+        margin-bottom: 0;
+      }
+
+      &:hover {
+        cursor: pointer;
+        text-decoration: none;
+      }
+      &__title {
+        color: $text-01;
```

Comment:
think we can remove this since we have it in the `themed-items` mixin already

Reply:
fixed.
Thanks